### ISSUE TITLE
DropdownMenu with custom Item is not handling focus correctly

### ISSUE THREAD

**Issue by vavdav** [radix-ui/themes#718]
I am using DropdownMenu and need to add one item with grid items - visual selector of items. I can do it with custom JS and CSS, but I cannot add accessibility. Is there any way how to allow you to navigate to those grid items via keyboard? I was thinking about Popover, but there I do not have sub-menu, toggle, or all the components that DropdownMenu offers.
